core/utils.js

```diff
@@ -12,9 +12,11 @@ welderApiScheme = welderApiScheme || 'http';
 function setupCockpitHttp() {
   const useHttps = welderApiScheme === 'https';
   const port = welderApiPort || (useHttps ? 443 : 80);
+  const superUser = welderApiPort ? "try" : undefined;
```

Comment:
Why is this conceptionally conditional on setting an explicit port, vs. using an implicit default?

Reply:
@martinpitt Should I always try for superuser?

Reply:
My gut feeling is that *if* it is legit to talk to welder as superuser, it should always be. Right now it should be; there once was some discussion about per-user recipes and such, which would be broken if "everybody" (by and large) talked to the API as root.

A better option would maybe be to check if the user already has read/write access to the Unix socket (through the `weldr` group), and only use `try` if not.

Reply:
The conclusion that I came to is that if we want to implement per-user recipes that we do that using branches and that it be enforced by the UI not by weldr. Currently nothing is using the branch support (it exists, but is likely not fully tested) so all blueprints are in the master branch.
So, as things stand now, there is no difference between accessing it as a member of the weldr group or as root.